### ISSUE TITLE
Фильтрация по значению поля

### ISSUE THREAD

**Issue by viacheslav594** [amocrm/amocrm-api-php#313]
Добрый день, коллеги.
Уточните, пожалуйста, в данный момент возможна ли фильтрация по значению поля (например дата == сегодня) или получить всех лидов, у которых выбран, к примеру один и тот же пункт в выпадающем списке?

Вроде видел это в альфе фильтрации, но не могу понять, как обстоит выкатывание? Потом вроде что-то стало доступно.

Вообще, как я понимаю фильтрация в альфе почти год?

**Comment by gotoroho** [comment]
Если фильтрация по CustomFieldValue вообще рабоатет - то прошу добавить в примеры фильтрацию например по номеру телефона, иначе не понятно какой массив значений передавать в функцию Filter->setCustomFieldsValues чтобы фильтрация работала как надо

**Comment by viacheslav594** [comment]
@bessudnov сориентируйте, пожалуйста?

**Comment by bessudnov** [comment]
Да, фильтрация всё еще в альфе, в конце мая будем подключать следующую большую пачку аккаунтов к функционалу. Заявки на подключение принимает техническая поддержка.

**Comment by bessudnov** [comment]
> setCustomFieldsValues

метод принимает на вход массив вида:
`[
    'ID ПОЛЯ' => 'ЗНАЧЕНИЕ ДЛЯ ФИЛЬТРАЦИИ'
]`

Например
`[
    4449124 => '+79169999999',
]`

Но чтобы фильтрация работала, аккаунт должен быть подключен к альфе через тех. поддержку

**Comment by viacheslav594** [comment]
@bessudnov, а есть сроки выхода из альфы? Уже год в альфе. И подключение оооочень долго, имея партнерку с амо, находясь в топ 10 интеграторов и регулярно публикуя виджеты в маркет ждал подключения к альфе с августа 20 по январь. Может быть возможно сделать переключатель в пользовательском интерфейсе в настройках и тд?

А есть/планируется ли возможность фильтрации поля (числа, даты) по интервалу? Например, дата до 01.01.2020 и тд?

**Comment by Supchck** [comment]
Коллегии так что на счёт фильтрации по кастимному полю. Все еще ни как ? Тех кккаунт подключил, метод так и не работает. 

Использую такой код $filter->setCustomFieldsValues([$this->amo_sourсe_field => 'value']);

**Comment by viacheslav594** [comment]
@Supchck там нужно оставлять заявку через тех поддержку на подключения, но рассмотрение, к сожалению, очень долгое - до полугода.

@bessudnov  сможете уточнить по сообщению выше?
